test/unit/logic/transaction.js

```diff
@@ -200,13 +223,31 @@ describe('transaction', function () {
 			expect(transaction.getBytes).to.throw();
 		});
 
-		it('should return same result when called multiple times', function () {
-			attachAllAssets(transaction);
+		it('should return same result when called multiple times (without data field)', function () {
 			var firstCalculation = transaction.getBytes(validTransaction);
 			var secondCalculation = transaction.getBytes(validTransaction);
 			expect(firstCalculation.equals(secondCalculation)).to.be.ok;
 		});
 
+		it('should return same result when called multiple times (with data field)', function () {
+			var transactionWithData = transaction.create(getValidTransactionData());
+			var firstCalculation = transaction.getBytes(transactionWithData);
+			var secondCalculation = transaction.getBytes(transactionWithData);
+			expect(firstCalculation.equals(secondCalculation)).to.be.ok;
+		});
+
+		it('should return same result of getBytes using /logic/transaction and list-js package (without data field)', function () {
+			var trsBytesFromLogic = transaction.getBytes(validTransaction);
+			var trsBytesFromLiskJs = node.lisk.crypto.getBytes(validTransaction);
+			expect(trsBytesFromLogic.equals(trsBytesFromLiskJs)).to.be.ok;
+		});
+
+		it('should return same result of getBytes using /logic/transaction and list-js package (with data field)', function () {
```

Comment:
@karmacoma This test case will fail since getBytes doesn't use data field we've added.